app/src/main/res/layout/activity_main.xml

```diff
@@ -9,7 +9,7 @@
     <TextView
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="Hello World!"
+        android:text="hogefuga"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/WataruSuzuki/Android-Example-Using-Danger/blob/94b8083d4bef629a9fac41baf782ccdd10e725fd/app/src/main/res/layout/activity_main.xml#L12"></span>Hardcoded string “hogefuga”, should use <code>@string</code> resource</td>
    </tr>
  </tbody>
</table>